inc/deploytask.class.php

```diff
@@ -179,13 +179,6 @@ public function title()
     */
     public function showActions($id)
     {
-
```

Comment:
I wonder if the whole `showActions()` method is still used, maybe should we drop it entirely. @stonebuzz ?

Reply:
Yes, I saw, but I do not even know if this code is stiall called/used... Anyway, I'll let it as is; removing dead call will make phpstan happy, that was the only goal :)